### ISSUE TITLE
Generic DoES poster?

### ISSUE THREAD

**Issue by Robotorium** [DoESLiverpool/somebody-should#1134]
We're having an open day at Sandbox on Saturday, and we'd like to have a poster advertising DoES. Jackie gave me a stack of the generic business cards, so we'll have those available, but I made this display thing where you can stick an A4 poster in and then there's a little box on the front to hold business cards, so it would be great if there's something available for that? Or do I just print off the logo and see how it goes?

**Comment by ajlennon** [comment]
@DoESsean had some business cards done recently I recall?

**Comment by ajlennon** [comment]
(Sounds a great idea!)

**Comment by Robotorium** [comment]
Yeah I've got business cards, I just need an A4 piece of paper that's about
DoES or has the logo on, to go in the stand. I can print it, I just don't
want to just decide what to put on it without input :)

Op ma 20 mei 2019 20:38 schreef Alex Lennon <notifications@github.com>:

**Comment by DoESsean** [comment]
I'll knock something up this week.

It'll probably be in a similar rough-and-ready style as my first pass at the poster in #1097, but that inspired others to improve it, so maybe this will do the same thing.

Aside from anything else, a generic DoES poster is on my list of things we need so this will be a good kickstart.

**Comment by DoESsean** [comment]
Right, there's a type of poster thing attached. It's pretty much as good as I can do with my 'skillset' and will suffice till people that have some talent see it and get wound up enough to fix it.

[Generic Poster.pdf](https://github.com/DoESLiverpool/somebody-should/files/3213183/Generic.Poster.pdf)

**Comment by DoESsean** [comment]
Also, there is a great design @zarino did for one of the pull up banners we take to events. That is in issue #677 and would be probably much better than my effort.

**Comment by ajlennon** [comment]
Love it @DoESsean - can we get some A5 (A4?) glossies printed up?

**Comment by DoESsean** [comment]
If no-one can do anything better - and I know @amcewen was saying it would do as something, rather than as something good ;) - then absolutely. Don't want to just rush in before good people can get their hands on it though!

**Comment by DoESsean** [comment]
So, this was last looked at a month ago.

One of the things we constantly say is that we need to be better at promoting ourselves, and having flyers and posters is a large part of that.

As no-one has actually said that they don't like the above effort, I think we get some printed and start getting them out there. If there are no objections by next week, I'll find a printer and make an order.

**Comment by ajlennon** [comment]
Hi everybody, @DoESsean 

I was chatting to my pal Lord Spesh about this. Turns out he's a graphic designer and has kindly had a go at a version for us. He's sent a first iteration over for comment. 

[A4 poster-2.pdf](https://github.com/DoESLiverpool/somebody-should/files/3389820/A4.poster-2.pdf)

**Comment by DoESsean** [comment]
Fantastic, thanks @ajlennon 

To be honest - and I'm aware I'm probably going to be shouted down here by people who know what they're actually talking about - it feels a bit sparse to me. I'm not sure if the white background would make it stand out. Although, of course, it could be precisely that that does make it stand out. What is your opinion?

**Comment by ajlennon** [comment]
I'm easy. Spesh put this forward as a first pass and is more than up for changes if it's felt that it would be helpful.

For me I'm no graphical design but I think less busy is generally better as a rule of thumb?

Any other voices out there?

For me I'd just like something to go and stick up around the Knowledge Quarter :))))))

**Comment by DoESsean** [comment]
That's pretty much what I think, just want to get something out there as quickly as we can - although it obviously needs to be right, and it's good if we can have a sort of uniform format that will cover multiple different posters/flyers. For some reason, that format also makes me think that the wording would need re-doing - it seems to fit better with the busier poster I did. Although I could well be wrong on that too.

@zarino, @Robotorium, @johnmckerrell, @amcewen, @jackie1050, @willcrobinson, @Helen-8, @cheapjack, as designers/directors/people who know more than I do, what would your opinions be.

**Comment by johnmckerrell** [comment]
It needed .com on the email I noticed

--

**Comment by DoESsean** [comment]
Yeah, there's definitely some proofing that needs doing, but that's not too big an issue for me just yet.
